Assignment/solution_student/index.html

```diff
@@ -1,35 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
 
-Sample Content
-——-
+  <head>
 
-ABOUT ME PAGE
+    <meta charset="UTF-8">
+    <title>Joe Johnson</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,600italic,700,700italic,800,800italic|Lora:400,400italic,700,700italic' rel='stylesheet' type='text/css'>
+  
+  </head>
+  
+  <header>
 
-I ENJOY LIFE AS A DEVELOPER
+    <div class="border"></div>
```

Comment:
Looks like this was something you tried out and then found a better solution to, which kudos to you for doing, and that's something that happens all the time in coding, but it's best to clean up that stuff that you end up not using whenever you remember :)